.changeset/breezy-comics-call.md

```diff
@@ -0,0 +1,6 @@
+---
+'@guardian/source-foundations': patch
+---
+
+- Text area, Text input, and Select active border colour persists when focused
+- Radio button and checkbox border changes on hover when unchecked and remains in the error state on hover when checked.
```

Comment:
It's good to be as explicit and concise as we can in the changelog. Using english descriptions rather than export names means the reader needs to read the english, understand it and then map it the export name (if they can remember it).

By referring to the export name directly, we can help them avoid that step:

```suggestion
- `TextArea`, `TextInput`, and `Select` active border colour persists when focused
- `Radio` and `Checkbox` border changes on hover when unchecked and remains in the error state on hover when checked.
```

Reply:
thank you, that’s much easier to read